.github/actions/slack-notification/README.md

```diff
@@ -0,0 +1,83 @@
+## About
```

Comment:
Can you add some information about the Slack handles mapping in vault and how to update it?

Reply:
docs were updated